tests/test_cli.py

```diff
@@ -491,7 +491,7 @@ def test_create(monkeypatch):
     data = {'SenzaComponents': [{'Config': {'Type': 'Senza::Configuration'}}],
             'SenzaInfo': {'OperatorTopicId': 'my-topic',
                           'Parameters': [{'MyParam': {'Type': 'String'}}, {'ExtraParam': {'Type': 'String'}}, {'DefParam': {'Type': 'String', 'DefaultValue': 'DefValue'}}],
-                          'StackName': 'test'}}
+                          'StackName': 'test', 'Tags': [{'CustomTag': 'CustomValue'}]}}
```

Comment:
tags have no order, so you should use a simple dict (instead of a list of dicts)